presets/gke.yaml

```diff
@@ -44,8 +44,6 @@ nginx-ingress:
         }'
     replicaCount: 2
     minAvailable: 1
-    nodeSelector:
```

Comment:
We should keep these and just update the docs.

Reply:
Those are redundant because are already configured in ` <<: *component_default`